src/utils/locale.ts

```diff
@@ -95,31 +143,54 @@ export const defaultNumberingSystem =
 export const getSupportedNumberingSystem = (numberingSystem: string): NumberingSystem =>
   isNumberingSystemSupported(numberingSystem) ? numberingSystem : defaultNumberingSystem;
 
-export function getSupportedLocale(locale: string): string {
-  if (locales.indexOf(locale) > -1) {
-    return locale;
-  }
+/**
+ * Gets the locale that best matches the context.
+ *
+ * @param locale – the BCP 47 locale code
+ * @param context - specifies whether the locale code should match in the context of CLDR or T9N (translation)
+ */
+export function getSupportedLocale(locale: string, context: "cldr" | "t9n" = "cldr"): SupportedLocales {
```

Comment:
Do you think we should use the type `SupportedLocales` here or anywhere else?

Reply:
we can add it for the `lang` property on each component but it's a global attribute. I don't think we can add it anywhere else. Am I missing anything?

Reply:
> we can add it for the lang property on each component but it's a global attribute. 

No, lets not do that.

> Am I missing anything?

Nope. Just wondering if anywhere else is applicable.